lib/vanagon/cli/dependencies.rb

```diff
@@ -0,0 +1,91 @@
+require 'docopt'
+require 'json'
+require 'vanagon/logger'
+
+class Vanagon
+  class CLI
+    class Dependencies < Vanagon::CLI
+      DOCUMENTATION = <<~DOCOPT.freeze
+        Usage:
+        dependencies [options] <project-name> <platforms>
+
+        Options:
+          -h, --help                       Display help
+          -c, --configdir DIRECTORY        Configuration directory [default: #{Dir.pwd}/configs]
+          -w, --workdir DIRECTORY          Working directory on the local host
+          -v, --verbose                    Only here for backwards compatibility. Does nothing.
+
+        Project-Name:
+          May be a project name of a project from the configs/projects directory or 'all' to generate dependencies for all projects.
+
+        Platforms:
+          May be a platform name of a platform from the configs/platforms directory or 'all' to generate dependencies for all platforms.
+
+      DOCOPT
+
+      def parse(argv)
+        Docopt.docopt(DOCUMENTATION, { argv: argv })
+      rescue Docopt::Exit => e
+        VanagonLogger.error e.message
+        exit 1
+      end
+
+      def run(options)
+        if Dir.exist?(File.join(options[:configdir], 'platforms')) == false ||
+           Dir.exist?(File.join(options[:configdir], 'projects')) == false
+
```

Comment:
Don't compare directly to `false`(or `true`). Ruby has a notion of `falsey` which includes `false` and `nil` which is implicit with `if` and `unless`. I suggest:

```ruby
platforms_directory = File.join(options[:configdir], 'platforms')
projects_directory = File.join(options[:configdir], 'projects')

unless Dir.exist?(project_directory) && Dir.exist?(platforms_directory)
  VanagonLogger.error (...)
end